.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release Core
+
+on:
+  push:
+    branches:
+    - disabled_master # rename to enable it
```

Comment:
For now this won't run unless you enable actions and rename this value to master.